pkgs/by-name/cy/cyanrip/package.nix

```
@@ -10,19 +10,30 @@
 , libmusicbrainz5
 , curl
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "cyanrip";
-  version = "0.9.0";
+  version = "0.9.2";
 
   src = fetchFromGitHub {
     owner = "cyanreg";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-gH/rWTRYX10Q2Y9oSaMu0bOy3SMbcSNmH3dkXHFAw90";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
```

```suggestion
    repo = pname;
    rev = "v${finalAttrs.version}";
```
appending pname surely should not change the download repo

also changing version here will have no effect until the hash is also changed

---

> appending pname surely should not change the download repo

Won't that require me to add `rec` back? I though the point of `finalAttrs` was to remove uses of `rec`. I can just hardcode it, I guess.

> also changing version here will have no effect until the hash is also changed

I did: `hash = "sha256-sWgHf8S4GZDAIvMUf5KvGy2y0JcUbRS53IjArdgokqc=";`